app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/dialogs/SendGeographicMessageDialog.java

```diff
@@ -75,12 +87,31 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    public void onSaveInstanceState(Bundle outState) {
+        super.onSaveInstanceState(outState);
+        outState.putParcelable(ARG_POINT_GEOMETRY, mPoint);
+    }
+
+    private void initSpinner() {
+
+        // Create an ArrayAdapter using the string array and a default spinner layout
+        ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(getActivity(),
```

Comment:
the ArrayAdapter.createFromResource return CharSequence. I have nothing to do about it.